substrate/frame/contracts-registry/src/migrations.rs

```diff
@@ -0,0 +1,98 @@
+use super::{pallet::Config, *};
+use frame_support::{pallet_prelude::*, storage_alias, traits::OnRuntimeUpgrade};
+use frame_system::pallet_prelude::BlockNumberFor;
+use sp_std::vec::Vec;
+
+#[cfg(feature = "try-runtime")]
+use sp_runtime::TryRuntimeError;
+
+pub mod v2 {
+	use super::*;
+
+	const TARGET: &'static str = "runtime::llm::migration::v2";
+    pub type I = ();
+
+	pub struct Migration<T, OldInterval>(sp_std::marker::PhantomData<(T, OldInterval)>);
+
+    #[storage_alias]
+    pub type PartiesSignatures<T: Config<I>> = StorageMap<
+        Pallet<T>,
+        Twox64Concat,
+        ContractIndex,
+        Vec<<T as frame_system::Config>::AccountId>,
+        OptionQuery,
+    >;
+	
+	#[storage_alias]
+	pub type JudgesSignatures<T: Config<I>> = StorageMap<
```

Comment:
The usual convention is to put aliases in some namespacing that identifies in which version they were correct. So we Either want `v1::JudgesSignatures` or `V1JudgesSignatures` here.

Right now we end up with using `v2::JudgesSignatures`, which suggests that it's storage definition for storage in version 2, but it's not.